src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_help.py

```diff
@@ -4,693 +4,685 @@
+    short-summary: Enable the Kubernetes addons in a comma-separated list.
+    long-summary: |-
+        These addons are available:
+            http_application_routing - configure ingress with automatic public DNS name creation.
+            monitoring - turn on Log Analytics monitoring. Uses the Log Analytics Default Workspace if it exists, else creates one. Specify "--workspace-resource-id" to use an existing workspace.
+            virtual-node - enable AKS Virtual Node (PREVIEW). Requires --subnet_name to provide the name of an existing subnet for the Virtual Node to use.
+  - name: --disable-rbac
+    type: bool
+    short-summary: Disable Kubernetes Role-Based Access Control.
+  - name: --enable-rbac -r
+    type: bool
+    short-summary: "Enable Kubernetes Role-Based Access Control. Default: enabled."
+  - name: --max-pods -m
+    type: int
+    short-summary: The maximum number of pods deployable to a node.
+    long-summary: If not specified, defaults to 110, or 30 for advanced networking configurations.
+  - name: --network-plugin
+    type: string
+    short-summary: The Kubernetes network plugin to use.
+    long-summary: Specify "azure" for advanced networking configurations. Defaults to "kubenet".
+  - name: --network-policy
+    type: string
+    short-summary: (PREVIEW) The Kubernetes network policy to use.
+    long-summary: |
+        Using together with "azure" network plugin.
+        Specify "azure" for Azure network policy manager and "calico" for calico network policy controller.
+        Defaults to "" (network policy disabled).
+  - name: --no-ssh-key -x
+    type: string
+    short-summary: Do not use or create a local SSH key.
+    long-summary: To access nodes after creating a cluster with this option, use the Azure Portal.
+  - name: --pod-cidr
+    type: string
+    short-summary: A CIDR notation IP range from which to assign pod IPs when kubenet is used.
+    long-summary: This range must not overlap with any Subnet IP ranges. For example, 172.244.0.0/16.
+  - name: --service-cidr
+    type: string
+    short-summary: A CIDR notation IP range from which to assign service cluster IPs.
+    long-summary: This range must not overlap with any Subnet IP ranges. For example, 10.0.0.0/16.
+  - name: --vnet-subnet-id
+    type: string
+    short-summary: The ID of a subnet in an existing VNet into which to deploy the cluster.
+  - name: --workspace-resource-id
+    type: string
+    short-summary: The resource ID of an existing Log Analytics Workspace to use for storing monitoring data. If not specified, uses the default Log Analytics Workspace if it exists, otherwise creates one.
+
+examples:
+  - name: Create a Kubernetes cluster with an existing SSH public key.
+    text: az aks create -g MyResourceGroup -n MyManagedCluster --ssh-key-value /path/to/publickey
+  - name: Create a Kubernetes cluster with a specific version.
+    text: az aks create -g MyResourceGroup -n MyManagedCluster --kubernetes-version 1.8.7
```

Comment:
Sorry, I missed one--could you update this to 1.12.6? Otherwise this looks good to me.

Reply:
Fixed